### ISSUE TITLE
workbench.editor.revealInActiveEditorGroup

### ISSUE THREAD

**Issue by schmeic** [microsoft/vscode#102824]
I would like an option to reveal an open editor in the active editor group - meaning, if an an editor is open for a file but the editor is in another editor group, first move it to the active editor group (assuming there is one), and then reveal the editor. Currently, if I set _"workbench.editor.revealIfOpen": true_, and then try and open a file while in an active editor group, the behavior is inconsistent - if the file is open in an editor, it opens in whatever editor group it happens to be in, but if the file is not open it will always open in the active editor group.

Basically, I want editors to work the same as emacs buffers.
